### ISSUE TITLE
Provide a good screenshot of a feelof the application on the README file

### ISSUE THREAD

**Issue by ocontant** [Icinga/icingaweb2-module-businessprocess#87]
Hi,

I would advice to give a good screenshot on the README page to have a rough idea of what the module does.  Even the documentation screenshot only provide one process being monitored and doesn't really show what it does. *Or I failed to understand it*. 

Having a few nodes and link between them to display what it does and how it display the information.   My understanding is that you want to monitor a complex service or infrastructure and not necessarily a single component or a single node that is part of this complex service.   Is my understanding right?   If not, we can try to improve the description. ;)

**Comment by Thomas-Gelf** [comment]
Hi @ocontant,

you're absolutely right. I'm already one day late with the release, main reason is the documentation and all those screenshots ;-) With every documentation step I discover something new to work on. Once I added the "main screenshot" to the README I'm all done :laughing:

Documentation will soon :tm: get a chapter showing a nested example and the related Check Command on CLI.

Cheers,
Thomas

**Comment by ocontant** [comment]
Awesome!  Can't wait to see it :)

Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Thomas Gelf <notifications@github.com> Date: 2017-01-26  11:45  (GMT-05:00) To: Icinga/icingaweb2-module-businessprocess <icingaweb2-module-businessprocess@noreply.github.com> Cc: Olivier Contant <contant.olivier@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [Icinga/icingaweb2-module-businessprocess] Provide a good screenshot of a feelof the application on the README file (#87) 
Hi @ocontant,
you're absolutely right. I'm already one day late with the release, main reason is the documentation and all those screenshots ;-) With every documentation step I discover something new to work on. Once I added the "main screenshot" to the README I'm all done 😆
Documentation will soon ™️ get a chapter showing a nested example and the related Check Command on CLI.
Cheers,

Thomas

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Icinga/icingaweb2-module-businessprocess","title":"Icinga/icingaweb2-module-businessprocess","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Icinga/icingaweb2-module-businessprocess"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Thomas-Gelf in #87: Hi @ocontant,\r\n\r\nyou're absolutely right. I'm already one day late with the release, main reason is the documentation and all those screenshots ;-) With every documentation step I discover something new to work on. Once I added the \"main screenshot\" to the README I'm all done :laughing:\r\n\r\nDocumentation will soon :tm: get a chapter showing a nested example and the related Check Command on CLI.\r\n\r\nCheers,\r\nThomas"}],"action":{"name":"View Issue","url":"https://github.com/Icinga/icingaweb2-module-businessprocess/issues/87#issuecomment-275440301"}}}
